core/templates/components/state-editor/state-editor-properties-services/state-name.service.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('State name service', () => {
     expect(sns.isStateNameEditorShown()).toBe(false);
   });
 
-  it('tests the get function', () => {
+  it('should test the get function', () => {
```

Comment:
should return the correct value for the getter functions*

Reply:
Done.